templates/test/dev/cluster-template-custom-builds-load.yaml

```
@@ -104,21 +108,110 @@ spec:
       owner: root:root
       path: /etc/kubernetes/azure.json
       permissions: "0644"
+    - content: |
+        #!/bin/bash
+
+        set -o nounset
+        set -o pipefail
+        set -o errexit
+        [[ $(id -u) != 0 ]] && SUDO="sudo" || SUDO=""
+
+        # Run the az login command with managed identity
+        if az login --identity > /dev/null 2>&1; then
+          echo "Logged in Azure with managed identity"
+          echo "Use OOT credential provider"
```

```suggestion
          echo "Using OOT credential provider"
```

Maybe? Seems more consistent with other log messages here.

---

This was copied over from the other custom build templates, I think changing it in all of them is out of scope of this PR.